docs/topics/migrations.txt

```diff
@@ -534,6 +534,31 @@ database migrations such as ``__init__()``, ``deconstruct()``, and
 which reference the field exist. For example, after squashing migrations and
 removing the old ones, you should be able to remove the field completely.
 
+.. _moving-model-between-apps:
+
+Moving a model between apps
+===========================
+
+  .. versionadded:: 5.0
+
+Django seamlessly handles the process of transferring models from one app
+to another. This involves recognizing the moved model and automatically
+generating the necessary migrations. The model's transition occurs within
+the project state, without affecting the database(unless required). Before
```

Comment:
```suggestion
the project state, without affecting the database (unless required). Before
```

I think this whole section might benefit from being the other way around; document that a table rename will happen unless the new model defines a `Meta.db_table` that matches the old one.